### DISCUSSION TITLE
Should the next release be labeled 7.0?

### DISCUSSION THREAD

**Discussion by ericjbohm** [UIUC-PPL/charm#3208] [General] [answered]
The substantial changes to the load balancing infrastructure and GPGPU programming, along with other changes, seem fundamental enough to warrant a major version number.

Arguments for or against?

**Comment by matthiasdiener** [comment]
Since there are substantial API changes, I think it makes sense to create a major release.

**Reply by minitu** [reply]
I agree with going 7.0 for the next release.

**Comment by matthiasdiener** [comment, marked as answer]
I think the consensus was to label the next release 7.0.
